src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/quick-start-nrql-condition.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Write your first NRQL alert condition
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+  - Alert conditions
+  - NRQL
+metaDescription: How to start writing NRQL alert conditions with a step by step user case.
+---
+
+Use [NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to create alert [conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/) with ease.
+
+Your condition only needs two attributes: a [query](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/#specify-data) and a [threshold](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/set-thresholds-alert-condition/).
+
+Follow our step-by-step example to write your first alert condition using the NRQL query builder.
+
+<Callout variant="important">
+This is a quick guide for creating NRQL alert conditions. For more information on NRQL alert examples, tips, and troubleshooting, see [Create NRQL alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/). 
+</Callout>
+
+## Step 1: Create your query
+
+Determine what alert condition you want to query before continuing. In our example, we want to create a condition that shows us each host’s CPU utilization. We write out the NRQL query as seen in the image below to track each host individually.
+
+![A screenshot of a NRQL query.](./images/NRQL-query-builder.png "A screenshot of a NRQL query.")
+
+If you need help writing in NRQL, see [NRQL syntax](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/#syntax).
+
+## Step 2: Create alert and establish your thresholds
+
+Your next step is to create an alert by clicking on the “Create alert” button below your query.
+
+![A screenshot of a NRQL query with an arrow pointing at the "Create alert" button.](./images/NRQL-create-alert.png "A screenshot of a NRQL query with an arrow pointing at the "Create alert" button.")
```

Comment:
I didn't see this button until I ran the query. Might be worth mentioning somewhere.

Reply:
The more I think about this image, the more I think that you can shift it up to replace this image with the one in Step 1. Click the "Create alert" button is the easiest part of this process. That might free you up to add some more useful info about ... I guess, "customizing the alerts condition"